### ISSUE TITLE
not usable on os x?

### ISSUE THREAD

**Issue by nilsbecker** [Armael/stone#5]
it seems stone depends on inotify which is linux-only. is this intended?

**Comment by Armael** [comment]
Right, at the moment it indeed depends on inotify to implement a "watch" feature (which automatically rebuilds whenever the sources are modified). 
To make it build under OS X, I see several options:

- Since the "watch" feature is not fundamental, the simplest solution would be to make it optional, and build without it on OS X
- Make the feature portable. I think dune implements a similar watch feature; it would be interesting to see how they do it in a portable way, and maybe reuse their code.

If you feel motivated implementing either of those options, I think that could be very helpful :)
